wazimap_ng/datasets/dataloader.py

```diff
@@ -9,7 +9,7 @@
 cache = {}
 def load_geography(geo_code):
     if geo_code not in cache:
```

Comment:
Why not change to uppercase before the if statement?

Reply:
Should I change it to uppercase before if statement ?